src/components/Aframe/my-loader.js

```diff
@@ -221,28 +220,34 @@ AFRAME.registerComponent("myloader", {
     // }
   },
 
-  updateTransfertexture: function () {
-    var imageTransferData = new Uint8Array(4 * 256);
-    for (var i = 0; i < 256; i++) {
-      imageTransferData[i * 4 + 0] = this.colorTransfer[i * 3 + 0];
-      imageTransferData[i * 4 + 1] = this.colorTransfer[i * 3 + 1];
-      imageTransferData[i * 4 + 2] = this.colorTransfer[i * 3 + 2];
-      imageTransferData[i * 4 + 3] = this.newAlphaData[i];
-    }
+  updateTransferTexture: function () {
+    if (
+      this.currentColorMap !== "" &&
```

Comment:
If you don't initialize `currentColorMap` as an empty string (above) this extra check isn't needed

Reply:
removed that check